etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-bricks.json

```diff
@@ -529,7 +529,7 @@
                 },
                 "cornerRadius": 0,
                 "datasource": "",
-                "description": "The Healing panel displays healing information for a given brick based on healinfo.  See https://gluster.readthedocs.io/en/latest/Troubleshooting/heal-info-and-split-brain-resolution/  \nfor how to troubleshoot healing and split brain issues",
+                "description": "The Healing panel displays healing information for a given brick based on Healinfo, i.e. statistics of entries pending heal, in split-brain and currently being healed.",
```

Comment:
what is "in split brain"? I think it should be split brain only.
Also, we don't show anything like "currently being healed". 
My suggestion: "`The Healing panel displays healing information for a given brick based on Healinfo, i.e. statistics of entries pending heal and split-brain.`"

Reply:
@cloudbehl yes, we do not show any thing like "currently being healed". What it means is that the status on the healing panel is calculated on the basis HealInfo which consists of statistics of entries pending heal, split-brain and currently being healed. 
@julienlim Can you please review this PR.

Reply:
@anmolsachan I am in-line with what @cloudbehl is suggesting. Otherwise it will be really confusing to the end-user. Let us lmit it to what is actually displayed in that panel

Reply:
Updated as per @cloudbehl 's suggestion.